src/css/toolbar.scss

```diff
@@ -0,0 +1,38 @@
+.toolbar {
+    padding:1rem;
+
+    .left-content {
```

Comment:
Wrong indentation

Reply:
Would you suggest how to correct this scss nesting indentation? Thanks <3

Reply:
the line bellow should have 1 more `tab` for indentation.